nodestream/pipeline/extractors/stores/aws/dynamodb_extractor.py

```diff
@@ -0,0 +1,199 @@
+from logging import getLogger
+from typing import Any, Dict, List, AsyncGenerator
+
+from ...credential_utils import AwsClientFactory
+from ...extractor import Extractor
+
+
+class DynamoObject:
```

Comment:
I believe Dynamo calls these values `attributes` so perhaps: 

```suggestion
class DynamoAttribute:
```